SecretAgent/SecretAgent/Global/Service/CoreDataManager.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  CoreDataManager.swift
+//  SecretAgent
+//
+//  Created by JiwKang on 2022/11/21.
+//
+
+import CoreData
+
+class CoreDataManager {
+    static let shared = CoreDataManager()
+    
```

Comment:
p2. 싱글톤 형태로 구현을 생각하신거라면

```suggestion
    static let shared = CoreDataManager()
    
    private init() { }
```
private init()을 추가해야 외부에서 새로운 인스턴스를 만드는 것을 방지할 수 있습니다..!

Reply:
우와.. 처음 알았습니다!!